projects/cashmere/src/lib/datepicker/calendar-body/calendar-body.component.html

```diff
@@ -0,0 +1,53 @@
+<!--
+  If there's not enough space in the first row, create a separate label row. We mark this row as
+  aria-hidden because we don't want it to be read out as one of the weeks in the month.
+-->
+<tr *ngIf="_firstRowOffset < labelMinRequiredCells" aria-hidden="true">
+    <td class="hc-calendar-body-label" [attr.colspan]="numCols" [style.paddingTop]="_cellPadding" [style.paddingBottom]="_cellPadding">
```

Comment:
I think we decided on our review call to remove the calendar body label because it's redundant with the dropdown above and will save us some vertical space.

Reply:
Looking at it, it didn't save us any vertical space but it was redundant so I removed it